Fixes compilation with gcc-3.4.0:

mntent_r.c: In function `__getmntent_r':
mntent_r.c:146: error: label at end of compound statement
mntent_r.c: In function `__addmntent':
mntent_r.c:214: warning: implicit declaration of function `alloca'

http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/misc/mntent_r.c.diff?r1=1.9&r2=1.11&cvsroot=glibc

===================================================================
RCS file: /cvs/glibc/libc/misc/mntent_r.c,v
retrieving revision 1.9
retrieving revision 1.11
diff -u -r1.9 -r1.11
--- libc/misc/mntent_r.c	1999/07/26 01:45:32	1.9
+++ libc/misc/mntent_r.c	2000/11/28 07:27:14	1.11
@@ -1,5 +1,5 @@
 /* Utilities for reading/writing fstab, mtab, etc.
-   Copyright (C) 1995, 1996, 1997, 1998, 1999 Free Software Foundation, Inc.
+   Copyright (C) 1995,1996,1997,1998,1999,2000 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -17,6 +17,7 @@
    write to the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
    Boston, MA 02111-1307, USA.  */
 
+#include <alloca.h>
 #include <mntent.h>
 #include <stdio.h>
 #include <string.h>
@@ -143,6 +144,7 @@
     case 1:
       mp->mnt_passno = 0;
     case 2:
+      break;
     }
   funlockfile (stream);
 
